Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -331,6 +331,12 @@ class BookmarksViewController: SiteTableViewController {
   
   fileprivate func configureCell(_ cell: UITableViewCell, atIndexPath indexPath: IndexPath) {
     
+    // Make sure Bookmark at index path exists,
+    // `frc.object(at:)` crashes otherwise, doesn't fail safely with nil
+    if let objectsCount = bookmarksFRC?.fetchedObjects?.count, indexPath.row >= objectsCount {
+        fatalError() // fail aggressively
```

Comment:
You can remove this comment, the fatalError should be enough, throw in a reason though:
`"Bookmarks FRC index out of bounds"`